src/testUtils/envVars.ts

```diff
@@ -9,10 +9,14 @@ export interface EnvVarSet {
 
 export const REQUIRED_ENV_VARS = {
   ENDPOINT_VERSION: '1.0.0',
-  POHTTP_TLS_REQUIRED: 'true',
   MONGODB_URI,
 };
 
+export const POHTTPH_CLIENT_REQUIRED_ENV_VARS = {
```

Comment:
Typo in "POHTTP".

More importantly, this variable should live in `testUtils/pohttpClient.ts`.